### PR TITLE
Mecha mmi Tweaks and Fix

### LINKED ISSUE #2708: Mechs with MMI pilots can't move
Unsure if this doesn't happen for posibrains too, since I've managed to get those to work before, but now the meaty MMIs are not able to move when in a mech. Happens with both combat and civilian varieties.

### PR DESCRIPTION
**Pull request by Crazylemon64**
* MMIs should now be able to move when in mechs again; a change in the baystation rigsuit port #2278 accidentally hijacked the else clause in brain.dm, which cut mech MMIs from moving
* MMI mechs can now have the MMI removed by removing the cell and using a crowbar. Since a pilot can restrict maintenance protocols, I see this as a consensual way to remove the MMI without trashing the exosuit. NOTE: Having used this a bit, I've noted that it might be possible to do on accident if you try to close the power hatch without screwing the cell in first. Should the tool be changed? The delay is currently 8 seconds, hopefully that's enough to note the error, and perhaps even serve as a good way to show the process exists to those who don't read changelogs
* MMIs in Oddyseus' now have a medical HUD
* You can cut apart mecha wreckage with a welding tool after all other components are removed
* Radio-enabled MMI's, and MMIs in a mech, can now use the built-in radio seamlessly with the standard "; [message]" format used by humans. This has removed the "broadcasting" verb from radio-enabled MMIs, as it is now obviated
* Mechs moving around now takes power again; the lack of which was introduced in #2171
* Posibrains inside of a mech can now continue to use and listen to Binary Chat
* Brains inside of an MMI or mech will now show station time and time to evac in the status panel, and MMI mechas will be shown their power level and suit integrity in the status screen as well.
* Fixes some annoying "hit by" messages that showed up when taking apart mecha wreckage
* MMIs that try to move while in someone's grasp no longer cause a runtime related to RIGsuits

Fixes #2708

### CODE REVIEW

File: code/modules/mob/living/carbon/brain/brain.dm [hunk 1]
```diff
@@ -54,7 +54,7 @@
 	if(in_contents_of(/obj/mecha))
 		canmove = 1
 		use_me = 1 //If it can move, let it emote
-	if(istype(loc, /obj/item/device/mmi))	canmove = 1 //mmi won't move anyways so whatever
+	else if(istype(loc, /obj/item/device/mmi))	canmove = 1 //mmi won't move anyways so whatever
```
Comment (Crazylemon64) [inline code/modules/mob/living/carbon/brain/brain.dm]: This is the line in particular that's causing problems: MMIs in mechs have been unable to move at all as of recent. Should I isolate this fix and put it in a separate PR?

**Comment by Spacemanspark** [comment]
Use dream daemon, log in with one account, go to Byond, lot out, log in with another account, and log that account into the game as well.

**Comment by Crazylemon64** [comment]
*ping

Being a mech MMI should be much nicer now; You now can use the radio in a non-painful manner, and you get a good heads-up on your condition as well in the status panel. Once review's done, this is merge-ready!

**Comment by Crazylemon64** [comment]
I've decided to test out a new item declaration syntax for the rods; I think it might be a good solution for stuff like walls deconstructing into many 1-item stacks, instead of a single stack with the needed quantity. Yeah or Neah?

**Comment by tigercat2000** [comment]
eehh
using the curly braces to modify new types is inconsistent, and barely documented

**Comment by Crazylemon64** [comment]
I've touched a little bit of RIGsuit code here, @tigercat2000, will the change I've made break anything?

**Comment by tigercat2000** [comment]
@Crazylemon64 looks fine to me, probably an oversight anyways

**Comment by Crazylemon64** [comment]
If there are no objections, this is ready to merge.
